### PR TITLE
corrected the nav-bar icon rotate bug

### PR DESCRIPTION
**Pull request by Sovrano28**
nav-bar icon is supposed to rotate on click not and not on hover. this was corrected.

### CODE REVIEW

File: src/components/Nav/Nav.jsx
Comment (PatMiracle) [inline src/components/Nav/Nav.jsx:1]: ## Using document.querySelector in React

You can use the querySelector and querySelectorAll within React. The downside it that you can lose the reference to the DOM element due to Reacts Reactivity. The solution is to use the useRef hook provided by react to store a reference to the DOM element.

**Review by PatMiracle — CHANGES_REQUESTED** [review]

File: src/components/Nav/Nav.jsx
Comment (Sovrano28) [reply src/components/Nav/Nav.jsx:1]: I'll correct that now. thanks @PatMiracle

**Comment by Sovrano28** [comment]
I have made the corrections, you like can review it @PatMiracle

**Review by PatMiracle — APPROVED** [review]

**Review by PatMiracle — APPROVED** [review]
## Good work
This is better. 🤝🏾

**Comment by PatMiracle** [comment]
## Nice one

I think we can move to making an interface for the chat center.

**Comment by PatMiracle** [comment]
## Nice one

I think we can move to making an interface for the chat center.

**Comment by Sovrano28** [comment]
let's go there! 😁💪🏾

**Pull request merged**